bigquery_etl/util/common.py

```diff
@@ -80,3 +84,40 @@ def write_sql(output_dir, full_table_id, basename, sql):
     with target.open("w") as f:
         f.write(sql)
         f.write("\n")
+
+
+class TempDatasetReference(bigquery.DatasetReference):
```

Comment:
Moving to `common` from `standard_args` since `standard_args` will eventually get removed once all scripts have been migrated to CLI (which will use click instead for argparse)